addons/mail/static/src/core/common/composer.scss

```diff
@@ -131,13 +131,41 @@
 }
 
 .o-mail-Composer-html {
-    padding-top: 10px; // carefully crafted to have the text in the middle in chat window
+    max-height: Min(400px, 60vh);
+    padding-top: 11px;
```

Comment:
Why is `Composer-html` overall style copy-paste of `Composer-inputStyle`?

I see differences here and there but it seems irrelevant: when I just remove this code and instead add `.o-mail-Composer-html` next to the `inputStyle` rule:

```scss
.o-mail-Composer-inputStyle, .o-mail-Composer-html {
   // current stylerules of 'inputStyle'
}
```
It looks and behaves the same stylistically. If there's no motivation for dedicated stylerules for `html`, then the DRYer code is to reuse `inputStyle` completely.

Reply:
because I think there was a discussion with seb before that we don't want to reuse the style of textarea (I was including o-mail-Composer-inputStyle as the classname), but i think eventually, we will remove Composer-inputStyle and Composer-input when we remove textarea so I think duplicating here is doable and easier to be removed in the future.